docs/howto/openstack/keystone/app-creds.md

```diff
@@ -0,0 +1,277 @@
+| expires_at   | 2024-08-31T23:59:59.000000                               |
+| id           | 43649f148bbe46ab8f1d745eea095db9                         |
+| name         | goofy_dijkstra                                           |
+| project_id   | dfc700467396428bacba4376e72cc3e9                         |
+| roles        | swiftoperator reader member creator load-balancer_member |
+| system       | None                                                     |
+| unrestricted | False                                                    |
+| user_id      | e50719d594f84f16ad13f88da540f762                         |
++--------------+----------------------------------------------------------+
+```
+
+No matter how you choose to list or view application credentials, secrets **are never** displayed.
+
+## Restricted vs unrestricted credentials
+
+By default, application credentials are created as being *restricted*; that is why in the `openstack application credential show` outout above, the `unrestricted` parameter is set to `False`.
+You cannot use restricted application credentials for Heat or Magnum, or for managing other application credentials or [trusts](https://docs.openstack.org/keystone/latest/user/trusts.html).
+If your application **has** to be able to perform such actions, and you accept the risks involved, you can always create *unrestricted* application credentials like this:
+
+```console
+$ openstack application credential create modest_mccarthy --unrestricted
+
++--------------+---------------------------------------------------------------+
+| Field        | Value                                                         |
++--------------+---------------------------------------------------------------+
+| description  | None                                                          |
+| expires_at   | None                                                          |
+| id           | c3505060e42542a29ce1b77435112590                              |
+| name         | modest_mccarthy                                               |
+| project_id   | dfc700467396428bacba4376e72cc3e9                              |
+| roles        | creator reader swiftoperator load-balancer_member member      |
+| secret       | qsV9G-qklvbZ8Fz5m7arghW7hHYDRCYWE3vim5dm3-EcXgQn-             |
+|              | pkvrDAOAW5nR7vfj32_HwtQbLjq0TyWM-gETg                         |
+| system       | None                                                          |
+| unrestricted | True                                                          |
+| user_id      | e50719d594f84f16ad13f88da540f762                              |
++--------------+---------------------------------------------------------------+
+```
+
+In the example output above, the `unrestricted` parameter is set to `True`.
+
+> The restricted-by-default policy acts as a safeguard, so compromised application credentials cannot be used for creating other sets of application credentials.
```

Comment:
Move this sentence up, so that it's part of the explanation of what differentiates restricted and unrestricted credentials (before the `--unrestricted` command example, that is).